src/phongo_atomic.c

```diff
@@ -0,0 +1,119 @@
+/*
```

Comment:
I suppose this and the corresponding header file were created from bson_atomic, with symbols renamed to have a `phongo_` prefix. As such, I trust our tests to ensure that this works properly and am not reviewing the logic contained. Let me know if my assumption is wrong.

Reply:
Correct. I reduced the file as much as possible to just keep the APIs necessary from our use case. So we've only got `int32_t` and the `int8_t` logic that requires. I did leave in the extra functions (e.g. "exchange") since they were all defined in the same block.